common/sagas/swap/orders.ts

```diff
@@ -185,10 +186,15 @@ export function* postShapeshiftOrderCreate(
       yield put(startPollShapeshiftOrderStatus());
     }
   } catch (e) {
-    const message =
-      'Connection Error. Please check the developer console for more details and/or contact support';
-    yield put(showNotification('danger', message, TEN_SECONDS));
-    yield put(shapeshiftOrderCreateFailedSwap());
+    if (e && e.message) {
+      yield put(showNotification('danger', e.message, TEN_SECONDS));
```

Comment:
Just putting this in the review for posterity: doesn't this case also need
```
yield put(shapeshiftOrderCreateFailedSwap());
```

Reply:
@wbobeirne i'll move it after the conditional but yes - i think you're right we need to call that on both cases.